src/wp-includes/class-wp-customize-setting.php

```diff
@@ -650,6 +658,11 @@ protected function get_root_value( $default = null ) {
 	protected function set_root_value( $value ) {
 		$id_base = $this->id_data['base'];
 		if ( 'option' === $this->type ) {
+			$option_validity = validate_option( $id_base, $value );
+			if ( is_wp_error( $option_validity ) ) {
+				return false;
+			}
+
```

Comment:
Is this right? Shouldn't the value have already been validated?

https://github.com/WordPress/wordpress-develop/blob/578e4569fb474e15490f10eee77b67429da5eb1e/src/wp-includes/class-wp-customize-manager.php#L2791-L2802